src/Workleap.DomainEventPropagation.Publishing/EventPropagationPublisherOptions.cs

```diff
@@ -13,4 +19,8 @@ public sealed class EventPropagationPublisherOptions
     public TokenCredential? TokenCredential { get; set; }
 
     public string TopicEndpoint { get; set; } = string.Empty;
+
+    public string TopicName { get; set; } = string.Empty;
```

Comment:
`TopicName` should be of type `string?`, considering it _can_ be `null` when `TopicType` is set to `TopicType.Default`

```suggestion
    public string? TopicName { get; set; }
```

Reply:
Good point